group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupConfigTest.java

```diff
@@ -165,6 +170,7 @@ private Properties createValidGroupConfig() {
         props.put(GroupConfig.SHARE_SESSION_TIMEOUT_MS_CONFIG, "45000");
         props.put(GroupConfig.SHARE_HEARTBEAT_INTERVAL_MS_CONFIG, "5000");
         props.put(GroupConfig.SHARE_RECORD_LOCK_DURATION_MS_CONFIG, "30000");
+        props.put(GroupConfig.SHARE_AUTO_OFFSET_RESET_CONFIG, "latest");
```

Comment:
Is there any test case which confirms beahviour with `earliet` offset.

Reply:
Thanks for the review. I will add another test just for validating the different available options for the config share.offset.auto.reset